WarehouseManagement.Core/Services/MarkerService.cs

```diff
@@ -162,57 +202,51 @@ public async Task<bool> IsDeletedByIdAsync(int id)
         return await repository.AllWithDeleted<Marker>().AnyAsync(m => m.Id == id && m.IsDeleted);
     }
 
-    public async Task<IEnumerable<MarkerDto>> GetDeletedMarkersAsync()
+    public async Task<IEnumerable<string>> GetDeletedMarkersAsync()
     {
         return await repository
             .AllWithDeletedReadOnly<Marker>()
             .Where(m => m.IsDeleted)
-            .Select(m => new MarkerDto
-            {
-                Name = m.Name,
-                Deliveries = m
-                    .DeliveriesMarkers.Select(dm => new MarkerDeliveryDto
-                    {
-                        DeliveryId = dm.DeliveryId,
-                        DeliverySystemNumber = dm.Delivery.SystemNumber
-                    })
-                    .ToList(),
-                Vendors = m
-                    .VendorsMarkers.Select(vm => new MarkerVendorDto
-                    {
-                        VendorId = vm.VendorId,
-                        VendorName = vm.Vendor.Name,
-                        VendorSystemNumber = vm.Vendor.SystemNumber
-                    })
-                    .ToList(),
-                Zones = m
-                    .ZonesMarkers.Select(zm => new MarkerZoneDto
-                    {
-                        ZoneId = zm.ZoneId,
-                        ZoneName = zm.Zone.Name
-                    })
-                    .ToList()
-            })
+            .Select(m => m.Name)
```

Comment:
There is no point of returning Mapping entities as you can't delete a Marker if it has some relations with Zone,Vendor or Delivery.